src/lib/err/torerr_sys.c

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2018, The Tor Project, Inc. */
+/* See LICENSE for licensing information */
+
+/**
+ * \file torerr_sys.c
+ * \brief Subsystem object for the error handling subsystem.
+ **/
+
+#include "orconfig.h"
+#include "lib/err/backtrace.h"
+#include "lib/err/torerr.h"
+#include "lib/err/torerr_sys.h"
+#include "lib/subsys/subsys.h"
+#include "lib/version/torversion.h"
+
+#include <stddef.h>
+
+static int
+torerr_subsys_init(void)
+{
+  configure_backtrace_handler(get_version());
```

Comment:
Shouldn't we keep the BUG that was removed and return -1 in case it fails?

Reply:
BUG isn't meaningful here -- without a backtrace handler, we can't bug.  But we can certainly return -1. Fixing.

Reply:
Ack.